packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1808,7 +1809,7 @@ export class ContainerRuntime extends EventEmitter
         this.submit(ContainerMessageType.FluidDataStoreOp, envelope, localOpMetadata);
     }
 
-    public async uploadBlob(blob: Buffer): Promise<IFluidHandle<string>> {
+    public async uploadBlob(blob: IsoBuffer): Promise<IFluidHandle<string>> {
```

Comment:
Consider it being a base64 string instead of buffer.
The way I'd chose between the two would be around performance.
If in all cases we use base64 on the wire, then I'd rather propagate it up to this layer and not do any conversions, as there is good chance that user code would also operate with base64 bits (that's the case with images), to avoid any back & forth conversions for nothing.
But if (SPO) storage (and fetch in general) allows binary data on the wire, then likely it's better to have binary all the way through to this layer, again to reduce number of conversions, because maybe there will be some usages that is based on binary data, not base64

Reply:
Yes, the new API allows binary data (`"octet-stream"`) on the wire

Reply:
Great!